include/dem/locate_local_particles.h

```diff
@@ -77,4 +76,42 @@ locate_local_particles_in_cells(
   typename dem_data_containers::dem_data_structures<
     dim>::particle_point_line_contact_info &particle_lines_in_contact);
 
+/**
+ * Updates the iterators to local particles in a map of particles
+ * (particle_container)
+ *
+ * @tparam dim Dimensionality of the geometry
```

Comment:
```suggestion
 * @tparam dim Dimensionality of the geometry which contains the particles
```
Here dim is really dimensionnality of the problem per se and not just the geometry. Geometries could be dim/spacedim if you had shells.